src/test/java/seedu/module/model/task/ModuleTest.java

```diff
@@ -44,67 +43,6 @@ public void isValidModule() {
         assertTrue(Module.isValidModuleFormat("DSA1101")); // alphabets only
     }
 
-    /**
-     * Constructs a string that is similar to the return message from getWorkloadCount() method in {@code Module}.
-     */
-    public String getWorkloadCountBuilder(int lowWorkloadCount, int mediumWorkloadCount, int highWorkloadCount) {
-        return String.format("Low workload tasks: %d, medium workload tasks: %d, high workload tasks %d\n",
-                lowWorkloadCount, mediumWorkloadCount, highWorkloadCount);
-    }
-
-    @Test
-    public void validIncrementWorkloadTest() {
-        Module originalModule = new Module("CS1101S");
-        originalModule.incrementWorkload(new Workload("3"));
-        String expectedMessage = getWorkloadCountBuilder(0, 0, 1);
-        assertEquals(originalModule.getWorkloadCount(), expectedMessage);
-
-        originalModule.incrementWorkload(new Workload("3"));
-        expectedMessage = getWorkloadCountBuilder(0, 0, 2);
-        assertEquals(originalModule.getWorkloadCount(), expectedMessage);
-
-        originalModule.incrementWorkload(new Workload("2"));
-        originalModule.incrementWorkload(new Workload("1"));
-        expectedMessage = getWorkloadCountBuilder(1, 1, 2);
-        assertEquals(originalModule.getWorkloadCount(), expectedMessage);
-    }
```

Comment:
think would need to add tests in ModuleManager instead later for the removal of these later